### PR TITLE
feat(310): map task v1 visuals, supporting both old and new api

### PR DESCRIPTION
**Pull request by anrusina**
Allow to support new MapTask structure, where individual logs are grouped by current phase status.

## Type
 - [ ] Bug Fix
 - [X] Feature
 - [ ] Plugin

## Are all requirements met?

 - [X] Code completed
 - [X] Smoke tested
 - [X] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Tracking Issue
fixes https://github.com/flyteorg/flyteconsole/issues/310

## Complete description

Map tasks completed before Admin update have old look:
Screen Shot 2022-04-07 at 3 20 13 PM
Task completed after admin Update have new look:
Screen Shot 2022-04-07 at 1 41 57 PM

Added Storybook implementations:
Screen Shot 2022-04-07 at 1 44 04 PM
In storybook below you can see all possible data shown. Log items rendered as lighter black text - are the ones that don't yet have logs links assigned
Screen Shot 2022-04-07 at 1 40 05 PM

Signed-off-by: Nastya Rusina <nastya@union.ai>

### CODE REVIEW

File: src/components/Executions/ExecutionDetails/NodeExecutionTabs/index.tsx [hunk 1]
```diff
@@ -17,6 +18,12 @@ const useStyles = makeStyles((theme) => {
     },
     tabs: {
       borderBottom: `1px solid ${theme.palette.divider}`,
+      '& .MuiTabs-flexContainer': {
+        justifyContent: 'space-around',
+      },
+    },
+    tabItem: {
+      margin: `0 ${theme.spacing(1)}px`,
```
Comment (anrusina) [inline src/components/Executions/ExecutionDetails/NodeExecutionTabs/index.tsx:26]: nit: update to `theme.spacing(0, 1, 0, 1)`

File: src/components/Executions/TaskExecutionsList/MapTaskExecutionListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    marginBottom: theme.spacing(2),
+  },
+}));
+
+interface MapTaskExecutionsListItemProps {
+  taskExecution: TaskExecution;
+  showAttempts: boolean;
+}
+
+const RenderOrder: TaskExecutionPhase[] = [
```
Comment (olga-union) [inline src/components/Executions/TaskExecutionsList/MapTaskExecutionListItem.tsx:38]: nit: switch to RENDER_ORDER for const

**Review by anrusina — COMMENTED** [review]
Sample how it runs: https://share.getcloudapp.com/2Nuzz7no

File: src/components/Executions/TaskExecutionsList/MapTaskExecutionListItem.tsx (hunk 2 above)
Comment (anrusina) [reply src/components/Executions/TaskExecutionsList/MapTaskExecutionListItem.tsx]: done

**Review by olga-union — APPROVED** [review]
🚢 🚢 🚢

**Pull request merged**